includes/ServiceWiring.php

```diff
@@ -9,7 +9,11 @@
 use MediaWiki\Extension\WikibaseReconcileEdit\Reconciliation\ItemReconciler;
 use MediaWiki\Extension\WikibaseReconcileEdit\Reconciliation\ReconciliationService;
 use MediaWiki\MediaWikiServices;
+use Wikibase\DataModel\Entity\Property;
 use Wikibase\DataModel\Services\Statement\GuidGenerator;
+use Wikibase\Lib\Store\Sql\Terms\CachedDatabasePropertyLabelResolver;
```

Comment:
The `CachedDatabasePropertyLabelResolver` is not yet working with Federated Properties. We tested it. We could build our own or make it work with wikibase. `CachedDatabasePropertyLabelResolver` is only used in client. What approach do you think we should take @lucaswerkmeister

Reply:
Let’s merge this without FP support and then see if we need it later.

Reply:
Sounds ok to me :)